### ISSUE TITLE
windows exe client 1.28.0 cannot sync with self host server.

### ISSUE THREAD

**Issue by Jecvay** [dani-garcia/vaultwarden#1924]
![image](https://user-images.githubusercontent.com/3610248/130250590-bdd93a44-eb83-473f-9431-cb8c2c321926.png)

but if I use exe client of version 1.27.1, everything is ok.

server info:

docker image: vaultwarden/server:latest

**Comment by BlackDex** [comment]
Confirmed, also have found the sollution

**Comment by praetorxyn** [comment]
I can confirm, and it's not just Windows.

I was using bitwardenrs/server-mysql, and I saw that the project had been renamed to vaultwarden, so I thought maybe I was just super out of date or something. So I copied my bitwarden database to vaultwarden, made a new vaultwarden user and gave it the appropriate permissions, commented out my bitwarden configuration YAML, and created a new vaultwarden config YAML and spun up the container using the new image, but I'm still seeing the issue.

This is what I am seeing in regards to a manual sync:
Web vault: Works fine
Brave extension 1.52.0: Works fine
Firefox extension 1.51.1: Works fine
Android app 2.11.3: Works fine
iOS app 2.11.1: Works fine
Windows client 1.28.0: Sync failed
macOS client 1.28.0: Sync failed
Linux client 1.28.0: Sync failed
